packages/application-shell/src/components/quick-access/create-commands.js

```diff
@@ -68,47 +64,54 @@ export default ({
         ].filter(Boolean),
       },
     project &&
-      featureToggles.canViewCategories &&
       hasSomePermissions(
-        [permissions.ViewProducts, permissions.ManageProducts],
+        [permissions.ViewCategories, permissions.ManageCategories],
         project.permissions
       ) && {
         id: 'go/categories',
         text: intl.formatMessage(messages.openCategories),
         keywords: ['Go to Categories'],
         action: { type: 'go', to: `/${project.key}/categories` },
         subCommands: [
-          {
+          hasPermission(permissions.ViewCategories, project.permissions) && {
             id: 'go/categories/list',
             text: intl.formatMessage(messages.openCategoriesList),
             action: { type: 'go', to: `/${project.key}/categories?mode=list` },
           },
-          {
+          hasPermission(permissions.ViewCategories, project.permissions) && {
```

Comment:
I decided to rather be repetitive here than trying to be smart.

Reply:
redudancy is king in this case